examples/classification/src/classificationSchema.ts

```diff
@@ -0,0 +1,8 @@
+export type Class = {
```

Comment:
Stick to either interfaces or type aliases for object types.

Also, I would not use the name `Class` here.

Reply:
Maybe just call it `Classification`